### PR TITLE
[cli] change 'binary' arg to '-x' for consistency (#5543)

### PR DESCRIPTION
**Pull request by jwhui**

### CODE REVIEW

**Review by abtink — APPROVED** [review]
Nice. LGTM. 👍

File: tools/harness-thci/#OpenThread.p# [hunk 1]
```diff
@@ -0,0 +1 @@
+≈ç
```
Comment (bukepo) [inline tools/harness-thci/#OpenThread.p#:1]: is this intended?

**Review by simonlingoogle — APPROVED** [review]
LGTM 👍

**Review by bukepo — APPROVED** [review]
LGTM:+1:

**Review by Irving-cl — APPROVED** [review]
LGTM👍

**Pull request merged**